rclcpp_components/src/node_main.cpp.in

```diff
@@ -37,7 +37,15 @@ int main(int argc, char * argv[])
   std::string class_name = "rclcpp_components::NodeFactoryTemplate<@component@>";
 
   RCLCPP_DEBUG(logger, "Load library %s", library_name.c_str());
-  auto loader = new class_loader::ClassLoader(library_name);
+  try {
+    auto loader = new class_loader::ClassLoader(library_name);
```

Comment:
this code does not compile.
you should have
`class_loader::ClassLoader loader = nullptr;` 
outside of the try catch

Reply:
slightly unrelated, but is there a reason why we use a raw pointer here?
this is not even deallocated at the end of the file (not a big deal, but still bad to see).